spec/fixtures/test_file_list_source_file.txt

```diff
@@ -0,0 +1,7 @@
+./spec/test1.rb
+spec/test2.rb[1]
+./spec/test3.rb[1:2:3:4]
+./spec/test4.rb:4
+./spec/test4.rb:5
+
+
```

Comment:
is there a double new line just to verify all the new lines are removed?

Reply:
yes, make sense?

Reply:
Ok. Just wanted to make sure I understand that we need a double new line on purpose here :) Thank you.

Reply:
I actually decided to only keep one line. sorry for the confusion.

Reply:
Ok. So let's keep 1 empty line at the end. I noticed you removed in a commit 2 lines.

Please let me know when you are done and then I can merge PR and release a new knapsack pro gem version for you.